src/main/kotlin/io/perfometer/dsl/DataDsl.kt

```diff
@@ -0,0 +1,30 @@
+package io.perfometer.dsl
+
+import io.perfometer.data.*
+import kotlin.reflect.KClass
+
+class DataProviderConfiguration<T : Any> {
+    var from: DataSource<T> = ListDataSource(emptyList())
```

Comment:
Do those properties have to be exposed as public mutable?

Reply:
Unfortunately yes. Otherwise you could not use them directly in "builder lambda" (extension function passed to `data` function)